From c0faa221bfb713b2287ed2f101285d56f28ba091 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Sat, 29 Jun 2019 12:56:02 +0800
Subject: [PATCH] do not support network setting for now

We have a long term plan to support it, but not now.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 pyanaconda/core/configuration/system.py | 3 ++-
 pyanaconda/installation.py              | 3 +++
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/pyanaconda/core/configuration/system.py b/pyanaconda/core/configuration/system.py
index 7459e2e..9b7ef96 100644
--- a/pyanaconda/core/configuration/system.py
+++ b/pyanaconda/core/configuration/system.py
@@ -135,7 +135,8 @@ class SystemSection(Section):
     @property
     def can_configure_network(self):
         """Can we configure the network?"""
-        return self._is_boot_iso or self._is_booted_os
+        # Do not support network config for now
+        return False
 
     @property
     def can_detect_unsupported_hardware(self):
diff --git a/pyanaconda/installation.py b/pyanaconda/installation.py
index 9eb40b3..0423a3d 100644
--- a/pyanaconda/installation.py
+++ b/pyanaconda/installation.py
@@ -79,6 +79,9 @@ def _writeKS(ksdata):
     auto_part_proxy = STORAGE.get_proxy(AUTO_PARTITIONING)
     auto_part_proxy.SetPassphrase("")
 
+    # Do not support network configuration
+    ksdata.network.network = []
+
     # Make it so only root can read - could have passwords
     with util.open_with_perm(path, "w", 0o600) as f:
         f.write(str(ksdata))
-- 
2.7.4

